index.html

```diff
@@ -10,6 +10,12 @@
     out in the same tree and use relative links so that they'll work offline.
    -->
 
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.24/css/jquery.dataTables.css">
+  
+  <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.24/js/jquery.dataTables.js"></script>
```

Comment:
With @OR13's change above, and SysTeam's installation, this should *hopefully* give us glorious sortable tables with resizable cursors!

```suggestion
  <link rel="stylesheet" type="text/css" href="https://www.w3.org/scripts/jquery-datatables/1.10/css/jquery.dataTables.css">
  
  <script type="text/javascript" charset="utf8" src="https://www.w3.org/scripts/jquery-datatables/1.10/js/jquery.dataTables.js"></script>
```

Reply:
Oops @TallTed, I was lousy, I did not realize that the data table script is also some external thing. Stupid me, but we have it now as one of the 'official' scripts that we can use. Which is great.

One more thing: is the result usable if scripts are disabled by the user? That may be an accessibility issue.

Reply:
@iherman - I'm not familiar enough with how datatables and jquery work to say.

@jricher - do you have any idea of whether they will fail gently or horribly if scripts are user-disabled?

Reply:
If scripts are disabled, then it turns into just a single, non-dynamic table, like it is today. DataTables works by taking an existing full table and adding dynamic controls to it for sort, filter, and paging. If those dynamic controls are never added, it simply shows up as a plain HTML table.